static/src/javascripts/projects/common/utils/$.js

```diff
@@ -0,0 +1,33 @@
+import bonzo from 'bonzo';
+import qwery from 'qwery';
+
+// Warning: side effect. This patches the bonzo module for use everywhere
+bonzo.aug({
+    height() {
+        return this.dim().height;
+    },
+});
+
+function $(selector, context) {
+    return bonzo(qwery(selector, context));
+}
+
+$.create = s => bonzo(bonzo.create(s));
+
+$.ancestor = (el, c) => {
+    if (el.nodeName.toLowerCase() === 'html') {
+        return false;
+    }
+    if (!el.parentNode || bonzo(el.parentNode).hasClass(c)) {
+        return el.parentNode;
+    }
+    return $.ancestor(el.parentNode, c);
+};
+
+$.forEachElement = (selector, fn) => {
+    const els = qwery(selector);
+    els.forEach(fn);
+    return els;
+};
```

Comment:
simplification looks good but I'd stick with `forEach` rather than `map` – you wouldn't expect a return value from a function called `forEachElement` so there's no need to build an array.